src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -136,17 +136,12 @@ public void sendBuildMessage(StepParameters stepParameters) {
         WebhookJobProperty property = (WebhookJobProperty) run.getParent().getProperty(WebhookJobProperty.class);
         if (property == null) {
             String webhookUrl = stepParameters.getWebhookUrl();
-            if (StringUtils.isBlank(webhookUrl)) {
-                listener.getLogger().println("No URL provided");
-                return;
-            }
-            try {
-                new URL(webhookUrl);
+            if (FormUtils.validateUrl(webhookUrl)) {
```

Comment:
Based on https://github.com/jenkinsci/office-365-connector-plugin/pull/24/files#diff-4d256ee5f164b70f36b78f29a615087cR177 the only reason to make here second validation is to make this code resistant for legacy code which allowed to save invalid URL such as empty string.

And still if this is to support legacy IMO it can be removed because it will try catch in worker